Source/WTF/wtf/PlatformHave.h

```diff
@@ -429,7 +429,7 @@
 #define HAVE_FOUNDATION_WITH_SAME_SITE_COOKIE_SUPPORT 1
 #endif
 
-#if PLATFORM(COCOA) || PLATFORM(GTK) || PLATFORM(WPE)
+#if PLATFORM(COCOA) || PLATFORM(GTK) || PLATFORM(HAIKU) || PLATFORM(WPE)
```

Comment:
Nice, I didn't know Haiku had dark mode support these days! (I should try out a recent build some day, last time was about a year ago 😇 )

Reply:
Yes, if you set your system colors to be dark (this is much easier to do in upcoming Haiku beta 5, we have simplified the Appearance preferences), WebKit will detect it and use dark mode when available